ui-kit/src/main/kotlin/org/p2p/uikit/components/edittext/v2/NewUiKitEditTextStyleAttrsApplier.kt

```diff
@@ -0,0 +1,86 @@
+package org.p2p.uikit.components.edittext.v2
+
+import androidx.core.content.res.getColorOrThrow
+import androidx.core.content.res.getDrawableOrThrow
+import androidx.core.content.res.getIntOrThrow
+import androidx.core.content.res.getResourceIdOrThrow
+import androidx.core.content.res.getStringOrThrow
+import android.content.res.ColorStateList
+import android.content.res.TypedArray
+import android.graphics.drawable.Drawable
+import android.text.method.DigitsKeyListener
+import org.p2p.uikit.R
+import org.p2p.uikit.databinding.WidgetUiKitEditTextNewBinding
+import org.p2p.uikit.utils.withImageOrGone
+import org.p2p.uikit.utils.withTextOrGone
+
+internal class NewUiKitEditTextStyleAttrsApplier {
+    private companion object {
+        private val ATTR_LABEL_TEXT = R.styleable.NewUiKitEditText_labelText
+        private val ATTR_HINT_TEXT = R.styleable.NewUiKitEditText_inputHintText
+        private val ATTR_INPUT_TEXT = R.styleable.NewUiKitEditText_inputText
+        private val ATTR_END_DRAWABLE = R.styleable.NewUiKitEditText_inputEndDrawable
+
+        private val ATTR_INPUT_TEXT_COLOR = R.styleable.NewUiKitEditText_inputTextColor
+        private val ATTR_INPUT_TEXT_STYLE = R.styleable.NewUiKitEditText_inputTextAppearance
+        private val ATTR_INPUT_BACKGROUND_TINT = R.styleable.NewUiKitEditText_inputBackgroundTint
+
+        private val ATTR_IS_INPUT_ENABLED = R.styleable.NewUiKitEditText_android_enabled
+        private val ATTR_INPUT_TYPE = R.styleable.NewUiKitEditText_android_inputType
+        private val ATTR_INPUT_DIGITS = R.styleable.NewUiKitEditText_android_digits
+        private val ATTR_NEXT_FOCUS_DOWN = R.styleable.NewUiKitEditText_android_nextFocusDown
```

Comment:
new edit text has it's own attributes. i decided not to use android: namespace for you to understand what can you do with this edit text using XML and app: namespace